aep/general/0127/aep.md.j2

```diff
@@ -1,5 +1,132 @@
 # HTTP and gRPC Transcoding
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/51) for more
-information.
+APIs that follow [resource-oriented design][resources] are defined using
+[RPCs][rpc], but the resource-oriented design framework allows them to also be
+presented as APIs that largely follow REST/JSON conventions. This is important
+in order to help developers use their existing knowledge: over 80% of the
+public APIs available follow most REST conventions, and developers are
+accustomed to that pattern.
+
+## Guidance
+
+APIs **must** provide HTTP definitions for each RPC that they define, except
+for bi-directional streaming RPCs, which can not be natively supported using
+HTTP/1.1. When providing a bi-directional streaming method, an API **should**
+also offer an alternative method that does not rely on bi-directional
+streaming.
+
+### HTTP method and path
+
+When using protocol buffers, each RPC **must** define the HTTP method and path
+using the `google.api.http` annotation:
```

Comment:
```suggestion
using the [`google.api.http`](https://buf.build/googleapis/googleapis/docs/main:google.api#google.api.Http) annotation:
```

Maybe worth making this a named link at the bottom, but it's hard to do that as a suggestion in GH.